sv_inline.h

```diff
@@ -206,7 +207,7 @@ static const struct body_details bodies_by_type[] = {
       + STRUCT_OFFSET(XPV, xpv_cur),
       SVt_PV, FALSE, NONV, HASARENA,
       FIT_ARENA(0, sizeof(XPV) - STRUCT_OFFSET(XPV, xpv_cur)) },
-
+/* sz_if_arena, copy_sz, ghost_sz, SVt, !upg, NV=0; , has_arena, arena_sz */
```

Comment:
I don't think it needs to be repeated, the time it takes to get that split view so you can see the first header (or the struct body_details definition) is trivial.
![image](https://github.com/user-attachments/assets/d7489981-ca3e-4abc-bf59-91928c449b88)